### PR TITLE
POC(): Demo usage of Penta styles in a react app

### LINKED ISSUE #9358: Consume Penta theming by getting style sheets from Core.
**Objective:**
Deliver a “demo app” that matches the core [demo](https://github.com/patternfly/patternfly/pull/5693) from the hackathon. 
This demo will be created by copying into core style sheets into workspace.

**End goal:**
Something like a page of an application - https://www.patternfly.org/v4/components/table/html-demos/sortable/
This is a visual demo, not going up for PR review.
image

Components that are showcased

- Brand
- Button
- Content
- Masthead
- Menu Toggle
- Nav
- Page
- Sidebar
- Table
- Toolbar

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
**What**: Closes #9358 

**Additional issues**:

Convenience link: https://patternfly-react-pr-9429.surge.sh/hackathon/hackathon/react-demos/hackathon/

### CODE REVIEW

**Comment by tlabaj** [comment]
@wise-king-sullyman  I am just wondering about the approach you took to pulling in the css.  Was there a reason we can not pull in the css for each component separately?

**Comment by wise-king-sullyman** [comment]
@tlabaj updated to use individual component css files rather than the combined file.

File: packages/react-docs/patternfly-docs/patternfly-docs.css.js
Comment (nicolethoen) [inline packages/react-docs/patternfly-docs/patternfly-docs.css.js:1]: Very cool

**Comment by andrew-ronaldson** [comment]
Just noticed this aria text on the context menu.
Screenshot 2023-08-22 at 10 54 08 AM

**Comment by wise-king-sullyman** [comment]
@andrew-ronaldson tooltips should be fixed now, the underlines seem to be how the styles are in the stylesheets I pulled in from core. 

If you want me to I can just edit that here and remove the underline though.

**Review by andrew-ronaldson — APPROVED** [review]
Looks great, Thanks!

**Review by srambach — APPROVED** [review]
Demo looks good to me!

**Comment by nicolethoen** [comment]
closing this PR since it was just a POC

**Pull request closed without merging**